templates/zerver/base.html

```diff
@@ -28,7 +28,7 @@
                 {% if filename.endswith(".css") -%}
                     <link href="{{ filename }}" rel="stylesheet" {% if csp_nonce %}nonce="{{ csp_nonce }}"{% endif %} />
                 {% elif filename.endswith(".js") -%}
-                    <script src="{{ filename }}" defer {% if csp_nonce %}nonce="{{ csp_nonce }}"{% endif %}></script>
+                    <script src="{{ filename }}" defer crossorigin="anonymous" {% if csp_nonce %}nonce="{{ csp_nonce }}"{% endif %}></script>
```

Comment:
@andersk can you review this commit, since it's a security setting?

Reply:
Thanks for that pointer!  Squashed such a change into the first commit.